clippy_lints/src/methods/unnecessary_filter_map.rs

```diff
@@ -52,15 +70,18 @@ pub(super) fn check<'tcx>(cx: &LateContext<'tcx>, expr: &'tcx hir::Expr<'tcx>, a
         } else {
             return;
         };
-        span_lint(
+        span_lint_and_sugg(
             cx,
             if name == "filter_map" {
                 UNNECESSARY_FILTER_MAP
             } else {
                 UNNECESSARY_FIND_MAP
             },
             expr.span,
-            format!("this `.{name}` can be written more simply using `.{sugg}`"),
+            format!("this `.{name}` can be written more simply"),
+            format!("try replacing with `.{sugg}`"),
```

Comment:
This doubles the suggestion in the output. Please reduce the text to "try instead" or "replace with".

Reply:
@llogiq fixed, thanks!